feature-libs/customer-ticketing/components/customer-ticketing-list/customer-ticketing-list.component.html

```diff
@@ -1,13 +1,176 @@
-<h2 class="cx-title d-none d-lg-block d-xl-block">
-  <!-- {{ 'title' | cxTranslate }} -->
-</h2>
+<ng-container *ngIf="tickets$ | async as ticketList">
+  <h3 class="cx-ticketing-list-title">
+    <span class="cx-ticketing-list-title-text">
+      {{ 'customerTicketingList.requestTitle' | cxTranslate }}
+    </span>
+    <span class="cx-ticketing-list-title-count">
+      ({{ ticketList.tickets.length }})
+    </span>
+  </h3>
 
-<ng-container>
-  <div class="cx-header-actions">
-    <!-- <cx-customer-ticketing-list-actions></cx-customer-ticketing-list-actions> -->
+  <div class="cx-ticketing-list-sort top">
+    <label class="cx-ticketing-list-form-group form-group">
+      <span>
+        {{ 'customerTicketingList.sortSubtitle' | cxTranslate }}
+      </span>
+      <cx-sorting
+        [sortOptions]="ticketList.sorts"
+        [sortLabels]="getSortLabels() | async"
+        (sortListEvent)="changeSortCode($event)"
+        [selectedOption]="ticketList.pagination.sort"
+        placeholder="{{ 'customerTicketingList.sortSubtitle' | cxTranslate }}"
+        [ariaLabel]="'customerTicketingList.sortOrders' | cxTranslate"
+        ariaControls="ticketing-list-table"
+      ></cx-sorting>
+    </label>
+    <div
+      *ngIf="ticketList.pagination.totalPages > 1"
+      class="cx-ticketing-list-pagination"
+    >
+      <cx-pagination
+        [pagination]="ticketList.pagination"
+        (viewPageEvent)="pageChange($event)"
+      ></cx-pagination>
+    </div>
   </div>
+  <ng-container *ngIf="ticketList.tickets?.length > 0; else noCustomerTickets">
```

Comment:
when it's empty, it's missing the title and count as seen in figma. `**Forget the sorting one (ignore it) as it doesn't make sense to display it when there's nothing. I'd like to see the title and count though**`